exp/term/ansi/wrap_test.go

```diff
@@ -149,19 +149,57 @@ var smartWrapCases = []struct {
 	{
 		name:     "long input2",
 		input:    "Rotated keys for a-good-offensive-cheat-code-incorporated/crypto-line-operating-system.",
-		expected: "Rotated keys for a-good-offensive-cheat-code-incorporated/crypto-line-operat\ning-system.",
```

Comment:
This test had me confused on what the expected result was. This was the test case my attempted fixes kept breaking (no pun intended) on.